CHANGELOG.md

```diff
@@ -121,6 +121,12 @@
   when there are too many regex routes and `router_flavor` is `traditional`.
   [#9624](https://github.com/Kong/kong/pull/9624)
 
+#### PDK
+
+- Extend `kong.client.tls.request_client_certificate` to support set CA certificate
```

Comment:
This is not accurate, we should call this "CA list hints". Otherwise it makes it sounds like the CA list is used for verification purpose.

Reply:
Indeed. Done.